python/rio-pmtiles/README.rst

```diff
@@ -0,0 +1,116 @@
+rio-pmtiles
+===========
+
+A plugin for the
+`Rasterio CLI <https://github.com/rasterio/rasterio/blob/main/docs/cli.rst>`__
+that exports a raster dataset to the PMTiles (version 3) format. Features
+include automatic reprojection and concurrent tile generation.
+
+This is derived from the original rio-mbtiles CLI by Sean Gillies at Mapbox. See CHANGELOG.md for differences.
```

Comment:
@sgillies please let me know if I am missing anything in attributing the project correctly. Though the other software in this repo is BSD I am keeping the MIT license for the `rio-pmtiles` portion as that was the original license.